src/SourceBuild/content/eng/pipelines/templates/steps/create-baseline-update-pr.yml

```diff
@@ -0,0 +1,40 @@
+parameters:
+# The pipeline that is being run
+# Used to determine the correct baseline update tool to run
+# Currently only supports "sdk" and "license"
+- name: pipeline
+  type: string
+
+- name: repo
```

Comment:
A comment would be helpful here indicating the expected format.  e.g. This is not a GitHub repo URL